### ISSUE TITLE
Cost reduction

### ISSUE THREAD

**Issue by raphaeltm** [defang-io/defang#248]
Right now the cost feels "off". I don't know if there's a good way around this, but I think this would be a real barrier to entry for small-mid sized projects. Tbh, if the costs grow with the same proportions, then I think larger projects as well:
![CleanShot 2024-04-04 at 15 27 00@2x](https://github.com/defang-io/defang/assets/910895/16eb2c33-4cfb-4b9d-8acc-5633c4f4ab1e)

I wouldn't be okay with spending so much on NatGateway hours.

**Comment by lionello** [comment]
Agree. We should only create a NATGW if you have a private service that needs internet access.
